lab1b/syntax.in

```diff
@@ -0,0 +1,29 @@
+<program> ::= <stmt>|<stmt><program> | <comment><program>
+<stmt> ::= <assign_stmt>;|<struct_stmt>|<io_stmt>;
+<struct_stmt> ::= <if_stmt>|<for_stmt>|<while_stmt>
+<io_stmt> ::= <read_stmt>|<print_stmt>
+
+<stmt_list> ::= <stmt> | <stmt><stmt_list>
+<stmt_block> ::= { <stmt_list> } | <stmt>
+
+<if_stmt> ::= if(<condition>) <stmt_block> | if(<condition>) <stmt_block> else <stmt_block>
+<while_stmt> ::= while(<condition>) <stmt_block>
+<for_stmt> ::= for(<identifier> in <identifier>) <stmt_block>
```

Comment:
also it seems that we need to define <identifier> here as well I think? not sure

Reply:
i have it in lexic